ossapi/ossapiv2.py

```diff
@@ -449,6 +449,25 @@ def beatmap(self, beatmap_id: int) -> Beatmap:
         """
         return self._get(Beatmap, f"/beatmaps/{beatmap_id}")
 
+    # /beatmapsets
+    # ____________
+
+    @request
+    def beatmapset_discussion_posts(self,
+        beatmapset_session_id: int = None,
+        limit: int = None,
+        page: int = None,
+        sort: SortOrder = None,
+        user: int = None,
+        with_deleted: bool = None
+    ) -> BeatmapsetDiscussionPostResult:
+        """
+        https://osu.ppy.sh/docs/index.html#get-beatmapset-discussion-posts
+        """
+        params = {"beatmapset_session_id": beatmapset_session_id,
+            "limit": limit, "page": page, "sort": sort, "user": user,
+            "with_deleted": with_deleted}
+        return self._get(BeatmapsetDiscussionPostResult, "/beatmapsets/discussions/posts", params)
```

Comment:
break over two lines to avoid going over 80 chars (I'm pretty strict about following this rule)